src/main/clojure/clara/rules/durability.clj

```diff
@@ -392,40 +392,35 @@
     (-> memory
         index-memory
         (update :memory
-                dissoc
+                ;; Assoc nil values rather than using dissoc in order to preserve the type of the memory.
+                assoc
                 ;; The rulebase does need to be stored per memory.  It will be restored during deserialization.
-                :rulebase
+                :rulebase nil
                 ;; Currently these do not support serialization and must be provided during deserialization via a
                 ;; base-session or they default to the standard defaults.
-                :activation-group-sort-fn
-                :activation-group-fn
-                :alphas-fn))))
+                :activation-group-sort-fn nil
+                :activation-group-fn nil
+                :alphas-fn nil))))
 
 (def ^:private create-get-alphas-fn @#'com/create-get-alphas-fn)
 
-(defn- opts->get-alphas-fn [rulebase opts]
+(defn opts->get-alphas-fn [rulebase opts]
   (let [fact-type-fn (:fact-type-fn opts type)
         ancestors-fn (:ancestors-fn opts ancestors)]
     (create-get-alphas-fn fact-type-fn
                           ancestors-fn
-                          rulebase)))
+                          (:alpha-roots rulebase))))
 
 (defn assemble-restored-session
   "Builds a Clara session from the given rulebase and memory components.  When no memory is given a new 
    one is created with all of the defaults of eng/local-memory.
    Note!  This function should not typically be used.  It is left public to assist in ISessionSerializer 
           durability implementations.  Use clara.rules/mk-session typically to make rule sessions.
-   
-   Options can be provided via opts.
-   These include:
-
-   * :fact-type-fn
-   * :ancestors-fn
-   * :activation-group-sort-fn 
-   * :activation-group-fn
 
    If the options are not provided, they will default to the Clara session defaults.
-   These are all described in detail in clara.rules/mk-session docs.
+   These are all described in detail in clara.rules/mk-session docs.  Note that the :activation-group-fn,
+   :activation-group-sort-fn, :fact-type-fn, and :ancestors-fn are attached to the rulebase, not the session,
```

Comment:
Right, I'm not trying to accept them, more calling out that the API has changed.  However, since is a *highly* experimental API perhaps you're right that calling it out specifically is just noise.